apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.html.ts

```diff
@@ -18,53 +24,57 @@ export function InstructorAiGenerateDraftEditor({
-      type: 'instructor',
-      page: 'course_admin',
-      subPage: 'questions',
-    },
-    options: {
-      hxExt: 'loading-states',
-    },
-    headContent: html`
-      ${compiledScriptTag('question.ts')}
-      <script defer src="${nodeModulesAssetPath('mathjax/es5/startup.js')}"></script>
-    `,
-    content: html`
-      <div class="mb-3">
-        <a href="${resLocals.urlPrefix}/ai_generate_question_drafts" class="btn btn-sm btn-primary">
-          <i class="fa fa-arrow-left" aria-hidden="true"></i>
-          Back to draft questions
-        </a>
-      </div>
-      <div class="card">
-        <div
-          class="card-header bg-primary text-white d-flex align-items-center justify-content-between"
-        >
-          Generate question with AI
-
-          <div>
-            <button
-              type="button"
-              class="btn btn-sm btn-light"
-              data-toggle="modal"
-              data-target="#finalizeModal"
-            >
-              <i class="fa fa-check" aria-hidden="true"></i>
-              Finalize question
-            </button>
-          </div>
-        </div>
-        <div class="card-body">
-          <div class="container">
-            <div class="row">
-              <div class="col-12 col-md-6">
-                <div>
-                  ${PromptHistory({ prompts, urlPrefix: resLocals.urlPrefix })}
+  // This page has a very custom layout, so we don't use the usual `PageLayout`
+  // component here. If we start building other similar pages, we might want to
+  // teach `PageLayout` how to render this kind of layout.
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        <meta
+          name="ace-base-path"
+          content="${nodeModulesAssetPath('ace-builds/src-min-noconflict/')}"
+        />
+        ${[
+          HeadContents({ resLocals }),
+          compiledScriptTag('question.ts'),
+          compiledScriptTag('instructorAiGenerateDraftEditorClient.ts'),
+          compiledStylesheetTag('instructorAiGenerateDraftEditor.css'),
+        ]}
+        <script defer src="${nodeModulesAssetPath('mathjax/es5/startup.js')}"></script>
+      </head>
+      <body hx-ext="loading-states">
+        <div class="app-container">
+          <div class="app-grid">
+            <div class="app-navbar">
+              ${Navbar({
+                navPage: 'course_admin',
+                navSubPage: 'questions',
+                resLocals,
+                marginBottom: false,
+              })}
+            </div>
+            <main id="content" class="app-content">
+              <div class="app-chat p-2 bg-light border-end">
+                <div class="app-chat-back mb-2">
+                  <a
+                    href="${resLocals.urlPrefix}/ai_generate_question_drafts"
+                    class="btn btn-sm btn-ghost"
+                  >
+                    <i class="fa fa-arrow-left" aria-hidden="true"></i>
+                    Back to draft questions
```

Comment:
Do we want to use the same style for this "Back to" button as we did for the similar button on the drafts page (primary button, I think)? And maybe separate it from the chat with a horizontal line or something?

At the moment this button kind of looks like it's part of the chat history because it's in basically the same style and position as the responses from the AI:
<img width="392" alt="image" src="https://github.com/user-attachments/assets/310421d0-3a86-4a4d-8517-3edc22668027" />

Reply:
I also think that the previous page is now more of a "AI question generation page" rather than a "drafts page"? I'm not in love with "Back to the AI question generation page" because it's super long. Maybe "Back to AI questions" or something?

Reply:
Done in f12063a. Here's how things look now:

<img width="916" alt="Screenshot 2025-02-04 at 09 49 58" src="https://github.com/user-attachments/assets/924afa12-f093-40a4-933b-ded5397ffe85" />

Reply:
Nice! I do worry that the Back link will be lost in a sea of similarly-colored links and tabs. This might not matter because people can always click on the main Questions link, and this might be improved with the new nav layout in any case.

Reply:
I very specifically don't want to make this a primary-colored button; there are already enough bright blue things on the page vying for one's attention, and it's unlikely that going back is going to be the primary action one will want to take on this page.

At any rate, going by my usual "if it's good enough for GitHub, it's good enough for us" metric, GitHub uses basically exactly the same style if you click through to an actions run from a PR:

<img width="403" alt="Screenshot 2025-02-04 at 12 13 40" src="https://github.com/user-attachments/assets/103e3f03-0f71-4314-bc60-754329d9d911" />